2015-Fall/lrusnac/CalculatorV2/CalculatorV2/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,361 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
You have a bunch of misplaced views in your UI. Paul showed how to see and fix this in lecture 1. Here it is not easy to see, but it will mean that the layout in interface builder and in the running application will differ, and you may have unexpected layouts. Check it out, or catch me tomorrow, and I can quickly show you.